packages/toast/README.md

```diff
@@ -64,32 +79,113 @@ import { Toast } from '@spectrum-web-components/toast';
 </sp-toast>
 ```
 
-## Variants
+#### Variants
+
+By default, the toast is rendered in gray and does not have an icon. This is used when the message is neutral in tone or when its semantics do not fit in any of the other variants.
+
+However, the toast supports several variants to convey different types of messages:
+
+<sp-tabs selected="negative" auto label="Toast Variants">
+<sp-tab value="negative" label="Negative"></sp-tab>
+<sp-tab-panel value="negative">
 
-### Negative
+Use `variant="negative"` to show an error or failure.
 
 ```html
 <sp-toast open variant="negative">
     This is negative information that you should read, soon.
 </sp-toast>
 ```
 
-### Positive
+</sp-tab-panel>
+<sp-tab value="positive" label="Positive"></sp-tab>
+<sp-tab-panel value="positive">
+
+Use `variant="positive"` to show a successful action or completion of a task.
 
 ```html
 <sp-toast open variant="positive">
     This is positive information that you should read, soon.
 </sp-toast>
 ```
 
-### Info
+</sp-tab-panel>
+<sp-tab value="info" label="Info"></sp-tab>
+<sp-tab-panel value="info">
+
+Use `variant="info"` to show an informative message.
 
 ```html
 <sp-toast open variant="info">
     This is information that you should read.
 </sp-toast>
 ```
 
-## Accessibility
+</sp-tab-panel>
+</sp-tabs>
+
+### Behaviors
+
+#### Timeout
+
+The toast can be configured to automatically dismiss itself after a specified duration. For accessibility reasons, the minimum timeout is 6000ms (6 seconds). For longer messages, it's recommended to add 1000ms for every 120 words.
+
+```html
+<sp-toast open timeout="6000">
+    This message will disappear after 6 seconds.
+</sp-toast>
+```
```

Comment:
nit: would be nice to have a reset button to see this in action, by the time I got to this section it has already timed out and just looked empty